### DISCUSSION TITLE
搭建个人 Memos 服务

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#28] [General]
# 搭建个人 Memos 服务

介绍一下自己是如何通过 telegram bot，GitHub Pages，cron task，Cloudflare R2，docker 等工具和代码自定义部署个人 memos 服务，也总结一下我自己对 memos 的简单思考

https://www.edony.ink/self-hosting-personal-memos-services/

**Comment by quanru** [comment]
推荐一个 Obsidian 插件，能够定时拉取 memos 到 Obsidian 的日记文件中，可以当做备份使用：https://github.com/quanru/obsidian-periodic-para

**Reply by edonyzpc** [reply]
谢谢推荐，我试试看

**Comment by marigold233** [comment]
我的memos打开好慢，我没用nginx反代，打开近30~40秒（呜呜呜

**Reply by quanru** [reply]
你用什么搭建的？试试这个，4天了，只花了我不到五毛钱 https://twitter.com/quanruzhuoxiu/status/1742905919481225693

**Reply by marigold233** [reply]
我用的阿里云的ecs，直接是IP方式访问的

**Reply by edonyzpc** [reply]
@marigold233 
可以使用 [Google Page Speed Insight](https://pagespeed.web.dev/) 分析看一下自己的服务性能，排除一下 ecs 性能带宽的影响，不过 memos 性能问题社区确实有抱怨过(参考 [issues](https://github.com/usememos/memos/issues?q=slow+OR+performance+))
<img width="1392" alt="image" src="https://github.com/edony-shadow-walker/comments/assets/7744664/ef80958c-9913-42ea-bf95-d154fafe08cc">

**Reply by quanru** [reply]
> 你用什么搭建的？试试这个，4天了，只花了我不到五毛钱 https://twitter.com/quanruzhuoxiu/status/1742905919481225693

数据库还是比较贵，转战 fly.io

**Reply by edonyzpc** [reply]
fly.io 是什么？类似 vercel 一样的东西

**Reply by marigold233** [reply]
fly.io 是不是需要 信用卡（

**Reply by quanru** [reply]
是的，需要一个 visa 信用卡：https://github.com/hu3rror/memos-on-fly/blob/main/README_no_litestream_zh-CN.md
